### PR TITLE
V1.6 updates

### PR DESCRIPTION
**Pull request by bennibbelink**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bennibbelink** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -15,9 +15,8 @@ fi
...
 ${PYTHON} install.py --prefix="${PREFIX}" \
-  --build_type="Release" \
-  --dont-allow-milps \
-  --deps-root="${PREFIX}" \
-  -DCMAKE_OSX_DEPLOYMENT_TARGET="${MACOSX_VERSION_MIN}" \
+  -DCMAKE_C_COMPILER=${GCC} \
+  -DCMAKE_CXX_COMPILER=${CXX} \
+  --allow-milps \
   ${skiprpath} \
-  --clean -j "${CPU_COUNT}"
```
Comment (gonuke) [inline recipe/build.sh:23]: Does `--clean` make a difference in this context? If so, then maybe best to keep it?

**Review by gonuke — CHANGES_REQUESTED** [review]
Question about building clean

File: recipe/build.sh (hunk 1 above)
Comment (bennibbelink) [reply recipe/build.sh:23]: We can leave it in for verbosity if you'd like, but it is building in a fresh environment so the flag should not be necessary.  It was removed in the Cyclus-feedstock as well

**Review by gonuke — APPROVED** [review]
Thanks @bennibbelink

**Pull request merged**